src/index.html

```diff
@@ -17,11 +17,333 @@
+        />
+      </a>
+
+      <nav class="header__nav">
+        <ul class="header__nav-list">
+          <li class="header__nav-list-item">
+            <a
+              href="#"
+              class="header__nav-list-item-link is-active"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="header__nav-list-item">
+            <a
+              href="#"
+              class="header__nav-list-item-link"
+            >
+              Samsung
+            </a>
+          </li>
+
+          <li class="header__nav-list-item">
+            <a
+              href="#"
+              class="header__nav-list-item-link"
+            >
+              Smartphones
+            </a>
+          </li>
+
+          <li class="header__nav-list-item">
+            <a
+              href="#"
+              data-qa="nav-hover"
+              class="header__nav-list-item-link"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="header__nav-list-item">
+            <a
+              href="#"
+              class="header__nav-list-item-link"
+            >
+              Gadgets
+            </a>
+          </li>
+
+          <li class="header__nav-list-item">
+            <a
+              href="#"
+              class="header__nav-list-item-link"
+            >
+              Tablets
+            </a>
+          </li>
+
+          <li class="header__nav-list-item">
+            <a
+              href="#"
+              class="header__nav-list-item-link"
+            >
+              Photo
+            </a>
+          </li>
+
+          <li class="header__nav-list-item">
+            <a
+              href="#"
+              class="header__nav-list-item-link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="catalog">
+      <div
+        class="card card--1"
+        data-qa="card"
+      >
+        <img
+          src="./images/imac.jpeg"
+          alt="imac"
+          class="card__image"
+        />
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <p class="card__code">Product code: 195434</p>
+        <div class="card__reviews">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__reviews-number">Reviews: 5</p>
+        </div>
+        <div class="card__price">
+          <p class="card__price-text">Price:</p>
+          <p class="card__price-cost">$2,199</p>
+        </div>
+
+        <button
+          class="card__link"
+          data-qa="card-hover"
+        >
+          buy
+        </button>
+      </div>
+
+      <div class="card card--2">
```

Comment:
Redundant, remove it everywhere
```suggestion
      <div class="card">
```